src/ripple/core/DatabaseCon.h

```diff
@@ -105,10 +105,16 @@ class DatabaseCon
         static std::unique_ptr<std::vector<std::string> const> globalPragma;
     };
 
+    struct CheckpointerSetup
+    {
+        JobQueue* jq;
+        Logs* logs;
+    };
+
```

Comment:
Instead of adding another setup struct to the ctor, consider moving this struct or ptrs into the already existing Setup struct.

Reply:
Fixed in `50335297d [fold] Longer variable names`

Reply:
@miguelportilla I'd rather keep the separate setup struct. There are two sets of constructors, one that takes a setup and one that does not. I'd like to make setting up a checkpointer the same for both types of constructors. The reason it's a struct instead of just two extra params, it I want to make it obvious at the caller that a checkpointer is being setup (because there's a local struct with `checkpointer` in its name).